### ISSUE TITLE
The question about that the pos weight of each anchor for each instance is normalized by the sum of all pos weights within the candidate bag.

### ISSUE THREAD

**Issue by MooresS** [strongwolf/DW#23]
Hello, I take the liberty to ask.
The pos weight of each anchor for each instance is normalized by the sum of all pos weights within the candidate bag.
```
p_pos_weight = (torch.exp(5*p_pos) * p_pos * center_prior_weights) / (torch.exp(3*p_pos) * p_pos * center_prior_weights).sum(0, keepdim=True).clamp(min=EPS)
```
In this code, why is the numerator ```5*p_pos ``` but the denominator ```3*p_pos```?

**Comment by strongwolf** [comment]
This setting mainly aims to adjust the balance between pos loss and neg loss. You can refer to this [issue](https://github.com/strongwolf/DW/issues/3).

**Comment by MooresS** [comment]
If so, is it normalized？Or it multiplies ```exp(2*p_pos)``` after normalized.

**Comment by strongwolf** [comment]
Yes, it can be seen that pos_weight multiplies an extra term after normalization.

**Comment by MooresS** [comment]
Thanks for your reply.
